locales/de.json

```
@@ -1466,7 +1454,7 @@
 				"howCanWeContactYou": "Wie können wir dich erreichen?"
 			},
 			"text": {
-				"ifTheEmailAddressIsNotTheCorrectOne": "Sollte die E-Mail-Adresse nicht die richtige sein, wende dich bitte an deinen {{shortTitle}} -Administrator.",
+				"ifTheEmailAddressIsNotTheCorrectOne": "Sollte die E-Mail-Adresse nicht die richtige sein, wende dich bitte an deinen {{shortTitle}} -Admin.",
```

```suggestion
				"ifTheEmailAddressIsNotTheCorrectOne": "Sollte die E-Mail-Adresse nicht die richtige sein, wende dich bitte an deinen {{shortTitle}} - Admin.",
```
i would add a space here as the minius is spaced from the title too

---

But the space has already been added.
Before: {{shortTitle}} -Admin
After: {{shortTitle}} - Admin
Or am I missunderstanding something?